lib/services/swagger-api-service.js

```diff
@@ -153,12 +153,14 @@ function swaggerFactory(
             res.set('Content-Type', 'application/json');
             if (_.isArray(res.body)) {
                 render = Promise.map(res.body, function(element) {
+                    element.basepath = req.swagger.operation.api.basePath;
```

Comment:
@tannoa2 Not a major concern, but you probably shouldn't be adding properties to the waterline record.  I'd suggest something like:
```javascript
var renderElement = _.merge({ basebase: req.swagger.operation.api.basePath }, element.toJSON());
```